Formula/guile-gnutls.rb

```diff
@@ -19,12 +20,10 @@ class GuileGnutls < Formula
   depends_on "guile"
 
   def install
-    # configure: WARNING: unrecognized options: --disable-debug
-    system "./configure", "--prefix=#{prefix}",
-                          "--with-guile-site-dir=#{share}/guile/site/3.0",
+    args = std_configure_args - ["--disable-debug"]
+    system "./configure", *args, "--with-guile-site-dir=#{share}/guile/site/3.0",
```

Comment:
Let's keep the original version; removing stuff from `std_configure_args` makes this needlessly confusing and harder to read.

Reply:
that is fair

Reply:
I can also just use `std_configure_args` as that `--disable-debug` is just a warning (which already happened for many other formulae)